base/exports.jl

```diff
@@ -1214,6 +1214,7 @@ export
     @__FILE__,
     @__DIR__,
     @__LINE__,
+    propagate_sourceloc!,
```

Comment:
There are some things that are public but not exported, usually it's whether they are documented in a public way as if to be used qualified. Most of those are under more specific modules than Base though, like Profile. Could maybe move this to the Meta module?